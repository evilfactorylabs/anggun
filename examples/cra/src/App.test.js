import "@testing-library/jest-dom/extend-expect";
import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

test('renders learn react link', () => {
  const { getByText } = render(<App />);
  const linkElement = getByText(/learn react/i);
  expect(linkElement).toBeInTheDocument();
});
test('I am the box', () => {
  const { getByTestId } = render(<App />);
    expect(getByTestId("button1")).toHaveTextContent(/i am the box/i)
    expect(getByTestId("button1")).toMatchSnapshot()
});

// if our want implement style assertation 
//
/*
 * @see {https://github.com/testing-library/jest-dom#tohavestyle}
  expect(container).toHaveStyle(`
          boxSizing: 'border-box';
                  display: flex;
                          flexWrap: 'wrap';
                                  width: '100%';
                                        `);
                                        */
