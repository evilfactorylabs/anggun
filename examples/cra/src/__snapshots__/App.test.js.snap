// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box with border radius 1`] = `
<div
  class="css-c0vxdr"
  data-testid="border-radius"
>
  Box with border radius
</div>
`;

exports[`I am the box 1`] = `
<div
  class="css-1p1esn3"
  data-testid="button1"
>
  I am the Box
</div>
`;
